Run a single unit test

lwassermann opened the pull request:
- Allow developers to run a single test file
- [The docs](https://mochajs.org/#the-test-directory) seem to recommend that way of changing the include path for nested test files.

*Note*: Since we're both using Mac OS, and the CI is using Unix, maybe somebody with Windows should also try whether this works. 🤔

Parnswir commented:
The problem with `--recursive` is, that it will also check `.js` files for mocha code, and there are some cases where a bunch of tests are spread over several files and then (sometimes conditionally) imported into one `.test.js` file. I think this is also why the Mocha test fail for this PR ;-)

lwassermann commented:
> `.test.js` file

There's an option for that as well. Updated the PR. 

> The problem with `--recursive` is, that it will also check `.js` files for mocha code, and there are some cases where a bunch of tests are spread over several files and then (sometimes conditionally) imported into one `.test.js` file. I think this is also why the Mocha test fail for this PR ;-)

Yes. And the `--extension` option does not seem to work 😞. I've reverted the package json changes and instead added another script.

pmeinhardt commented on README.md:
```diff
@@ -50,6 +50,8 @@ For more details how to set up Visual Studio Code, read [this document](https://
 1. Go into project folder
 2. run `npm run test`
 
+To run a single test, use `npm run mocha-single <path/to/unit.test.js>`.
```
I think this needs to be slightly adjusted:

```suggestion
To run a single test, use `npm run mocha-single -- <path/to/unit.test.js>`.
```

— https://docs.npmjs.com/cli/run-script#description

lwassermann replied:
🤔 Interesting. It works without the `--`. I'll add it to the recommendation nevertheless, to make it more explicit.

pmeinhardt commented:
Sweet. This would be pretty useful to have in `develop`. 🍰
